docs/developer/release/7-update-deployment-tools.md

```diff
@@ -0,0 +1,51 @@
+# Update Deployment Tools
+
+We must update [grafana/deployment_tools](https://github.com/grafana/deployment_tools) (unless it is already in the desired state) to test our RCV or to stop testing it.
+
+## Steps
+
+1. Create a branch from `main` for [grafana/deployment_tools](https://github.com/grafana/deployment_tools).
+
+2. Open a PR to update [main.jsonnet](https://github.com/grafana/deployment_tools/blob/master/ksonnet/environments/grafana-agent/main.jsonnet).
```

Comment:
Ditto on exposed URLS

Reply:
Good catch, I think we remove this from the repo as documentation relevant for internal grafana only.